app/src/main/java/org/p2p/wallet/striga/sms/StrigaSmsInputInteractor.kt

```diff
@@ -93,6 +93,21 @@ class StrigaSmsInputInteractor(
     }
 
     suspend fun resendSms(): StrigaDataLayerResult<Unit> {
+        if (inAppFeatureFlags.strigaSmsVerificationMockFlag.featureValue) {
+            smsInputTimer.startSmsInputTimerFlow()
+            return if (smsInputTimer.smsResendCount > 5) {
+                StrigaDataLayerError.ApiServiceError(
+                    response = StrigaApiErrorResponse(
+                        httpStatus = 400,
```

Comment:
const ?

Reply:
Greate point! I'll extract the entire error result to file variable, as it duplicated multiple times